docs/provisioning-methods/zte.md

```diff
@@ -24,28 +24,27 @@ To enter the DPC extras, navigate to the Esper Console.
 
 ![download](./images/ZTE/3-downloadCOnfig.png)
 
-Login to the Esper Console and navigate to the Device and Group section. Click on a QR code icon assosiated with any group and download the config. Copy the following contents from the downloaded Config file and paste it into the DPC extras. Click ADD.
-
+Login to the Esper Console and navigate to the Device and Group section. Click a QR code icon associated with the group you want to configure with ZTE and click the “Download Config” button at the bottom of the pop-up window with your QR code. Open the downloaded JSON config file, and you will need to copy the following contents: 
+
+**Note:** In the downloaded config file, there will be a section of the code that is not needed. Copy only the section below, and use a formatter such as [JSON formatter](https://jsonformatter.curiousconcept.com/) to ensure the code is correctly formatted.
+
+```json
```

Comment:
@radhak21 please format the code yourself as well, the spaces and alignment as of now is not correct